.github/workflows/clippy-fmt.yml

```diff
@@ -37,6 +37,10 @@ jobs:
 
       - run: cargo clippy --all-targets
 
+      - run: cargo check --release
+        env:
+          RUSTFLAGS: -D warnings
```

Comment:
i don't think it's worth to increase CI times for this

Reply:
but we actually didn't notice the warnings because we usually use debug build for development. And users will face the warnings because they build wry in release mode for building their application.

Regarding to the increase, this step only takes 10 seconds on CI and I feel it's worth.

Reply:
you'll only see the warnings if you try to build wry from source instead of the published crate

Reply:
let me guess.. `RUSTFLAGS="-D warnings"` comes from https://github.com/actions-rust-lang/setup-rust-toolchain ? I hate that 😂